src/onegov/form/validators.py

```diff
@@ -344,6 +344,43 @@ def __call__(self, form: 'Form', field: 'Field') -> 'Form | None':
         return parsed_form
 
 
+class ValidSurveyDefinition(ValidFormDefinition):
+    """ Makes sure the given text is a valid onegov.form definition for
+        surveys.
+    """
+
+    def __init__(self, require_email_field: bool = False):
+        super().__init__(require_email_field)
+
+    invalid_field_type = _("Invalid field type for field '{label}'. Please "
+                           "use the plus-icon to add allowed field types.")
+
+    def __call__(self, form: 'Form', field: 'Field') -> 'Form | None':
+        from onegov.form.fields import UploadField
+
+        parsed_form = super().__call__(form, field)
+        if parsed_form is None:
+            return None
+
+        # Exclude fields that are not allowed in surveys
+        errors = None
+        for field in parsed_form._fields.values():
+            if isinstance(field, (UploadField, DateField, TimeField,
+                                  DateTimeLocalField)):
+                error = field.gettext(self.invalid_field_type.format(
+                    label=field.label.text))
```

Comment:
```suggestion
    invalid_field_type = _("Invalid field type for field '${label}'. Please "
                           "use the plus-icon to add allowed field types.")

    def __call__(self, form: 'Form', field: 'Field') -> 'Form | None':
        from onegov.form.fields import UploadField

        parsed_form = super().__call__(form, field)
        if parsed_form is None:
            return None

        # Exclude fields that are not allowed in surveys
        errors = None
        for field in parsed_form._fields.values():
            if isinstance(field, (UploadField, DateField, TimeField,
                                  DateTimeLocalField)):
                error = field.gettext(self.invalid_field_type %
                    {'label': field.label.text})
```
This is the proper way to do dynamic string interpolation with translation strings. Your way would've worked too but it was backwards, you needed to format after calling `gettext`, not before, otherwise it remain untranslated because the interpolated translation string does not exist. Remember to update the translations with the missing $ as well.